core/src/main/java/org/jboss/pnc/causeway/ctl/PncImportController.java

```diff
@@ -23,5 +23,5 @@
  */
 public interface PncImportController {
 
-    public void importMilestone(int milestoneId, CallbackTarget callback, String callbackId);
+    void importMilestone(int milestoneId, CallbackTarget callback, String callbackId);
```

Comment:
Why to remove the public modifier in file that you don't change otherwise? It has the same meaning.